src/NzbDrone.Core.Test/ParserTests/SeasonParserFixture.cs

```diff
@@ -87,6 +87,7 @@ public void should_parse_partial_season_release(string postTitle, string title,
         [TestCase("Series Title S01 - S07 BluRay 1080p x264 REPACK -SacReD", "Series Title", 1)]
         [TestCase("Series Title Season 01-07 BluRay 1080p x264 REPACK -SacReD", "Series Title", 1)]
         [TestCase("Series Title Season 01 - Season 07 BluRay 1080p x264 REPACK -SacReD", "Series Title", 1)]
+        [TestCase("Series Title Complete Series S01 S04 (1080p BluRay x265 HEVC 10bit AAC 5.1 Vyndros)", "Series Title", 1)]
```

Comment:
Please add a case for just a space between the seasons.

Reply:
added two more cases

Reply:
The test case with "S01 S04", is it supposed to recognize it as seasons 1 through 4, or as seasons 1 and 4?  Because I would think it would be 1 and 4.  I wouldn't think that specific one would show up, more likely they would be consecutive seasons, of course.

Reply:
Sonarr expects either consecutive seasons or a range of inclusive seasons